<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hufcrypt GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="grpFile">
      <property name="title">
       <string>파일</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelInput">
         <property name="text">
          <string>입력 파일</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leInput"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnBrowseIn">
         <property name="text">
          <string>찾기…</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelOutput">
         <property name="text">
          <string>출력 파일</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leOutput"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnBrowseOut">
         <property name="text">
          <string>찾기…</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hboxModePw">
      <item>
       <widget class="QGroupBox" name="grpMode">
        <property name="title">
         <string>모드</string>
        </property>
        <layout class="QHBoxLayout" name="layoutMode">
         <item>
          <widget class="QRadioButton" name="rbEncrypt">
           <property name="text">
            <string>암호화</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbDecrypt">
           <property name="text">
            <string>복호화</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpPassword">
        <property name="title">
         <string>비밀번호</string>
        </property>
        <layout class="QHBoxLayout" name="layoutPw">
         <item>
          <widget class="QLineEdit" name="lePassword">
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPwToggle">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string extracomment="비밀번호 보기"/>
           </property>
           <property name="text">
            <string>···</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="progress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="logBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>실행</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>328</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
